### DISCUSSION TITLE
Create nodes synchronization without input and outputs depency

### DISCUSSION THREAD

**Discussion by acnazarejr** [kedro-org/kedro#816] [Idea]
I have two nodes. The first (NODE A) creates a table on my data warehouse and the second (NODE B) consumes this table. However, the first node (A) outputs isn't an input for the second node (B). 

So, it's possible to force the NODE B to wait for the NODE A execution to finish, when I use the ParallelRunner?

**Comment by antonymilne** [comment]
I believe this is not (easily) possible without forcing the order of execution by defining dependency between the two nodes with a dataset. Note that you don't have to make this dataset the actual real table in question; you can make a new dummy one:
```yaml
# catalog.yml
dummy_dataset:
  type: MemoryDataSet
```

And then define your nodes:
```python
node(node_A, inputs, outputs=["dummy_dataset"])
node(node_B, inputs=["dummy_dataset"], outputs) 
```

**Reply by idanov** [reply]
The catalog entry is not even needed, since it will be created by default from Kedro.
